Constants/Constants.go

```diff
@@ -0,0 +1,4 @@
+package constant
```

Comment:
The package name is different from than actual package.
constants -> constant

It should be constants.